lrpar/src/lib/ctbuilder.rs

```diff
@@ -1189,13 +1193,11 @@ pub struct CTParser<StorageT = u32>
 where
     StorageT: Eq + Hash,
 {
+    src: String,
```

Comment:
I don't think we want to store this. If the user wants to store it, that's up to them, but we don't even make use of it.

Reply:
The issue is that the user doesn't have it, it is currently local to build, so if we don't return it in some way the user can never get to it.

Reply:
I pushed a proposed fix in [eafabc4](https://github.com/softdevteam/grmtools/pull/347/commits/eafabc42bc04f20f4022b4faa240b39add9fd23a) allowing the user to specify a buffer so they'll own it.

Reply:
Just wanted to note down one thing (not caused by) but brought to my attention due to the removal here, is the lack of any form of `SrcIdx` or anything which can `Eq` test whether a src file belongs to a `Spanned`.  Which will become more apparent if we coalesce errors across lex/yacc.

Reply:
Now that's an interesting question! I guess this is disambiguated if we have `GrammarError` vs `LexerError`, but you're right that we have to guarantee that there is at least *some* way of disambiguating things!